### ISSUE TITLE
Unable to save sheet to JSON if column is datetime

### ISSUE THREAD

**Issue by layertwo** [saulpw/visidata#179]
I am currently unable to save a sheet to JSON (or JSONL) if I have a column set as a datetime object. Saving as .tsv or .csv works correctly.

Here's the error I'm getting:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/visidata/vdtui.py", line 643, in toplevelTryFunc
    ret = func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/visidata/loaders/json.py", line 100, in save_jsonl
    fp.write(jsonenc.encode(rowdict) + '\n')
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python3.7/site-packages/visidata/loaders/json.py", line 78, in default
    return json.JSONEncoder.default(self, o)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type date is not JSON serializable
```

**Comment by saulpw** [comment]
Good catch, @lmessenger.  A fix is in the works.

**Comment by saulpw** [comment]
Fixed in 674d8f88a635beaa70a66d98159c8184c3a3a1fb.  Will probably be in 1.3.1.
